yang/vyatta-protocols-frr-isis-v1.yang

```diff
@@ -730,5 +734,7 @@ module vyatta-protocols-frr-isis-v1 {
   augment /if:interfaces/interfaces-loopback:loopback/interfaces-loopback:ipv6 {
     uses itf-ip-router-isis;
   }
-
+  augment /if:interfaces/interfaces-switch:switch/interfaces-switch:vif/interfaces-switch:ip {
+    uses itf-ip-router-isis;
```

Comment:
Please can you also augment `itf-ip-router-isis` under the ipv6 container, and `isis` under `vif` (ie. to match the other interface types).

Sorry, I missed this on the first pass.

Reply:
Resolved. That was missed.
Thanks.